.github/actions/deploy-release-candidate-v1/README.md

```diff
@@ -0,0 +1,37 @@
+# deploy-release-candidate-v1
+
+A GitHub Action to prepare already created release candidate for QA.
+
+## Inputs
+
+| Name            | Required | Description                                                | Default |
+| --------------- | -------- | ---------------------------------------------------------- | ------- |
+| `version`       | Yes      | A version of the release candidate                         | NA      |
+| `revision-url`  | Yes      | A commit revision URL                                      | NA      |
+| `owner`         | Yes      | An owner of the repository                                 | NA      |
+| `repo`          | Yes      | A repository name                                          | NA      |
```

Comment:
This should use the same wording and requirements as https://github.com/dequelabs/axe-api-team-public/blob/17d61714dab39c92c80e2a6b6413e0ab11d70a41/.github/actions/get-release-issue-v1/README.md?plain=1#L11

```suggestion
| `repo`    | No       | The repository e.g. dequelabs/axe-core-npm, then supply "axe-core-npm"           | `github.context.repo.repo`  |
```